module/platform/OpenCR/HardwareIO/src/HardwareIO.cpp

```diff
@@ -13,7 +13,7 @@
 #include "utility/math/comparison.hpp"
 #include "utility/support/yaml_expression.hpp"
 
-#define DEBUG_ENABLE_BUTTON_SPOOF 1
+#define DEBUG_ENABLE_BUTTON_SPOOF 0
```

Comment:
I feel like this is out of scope for this PR and also we should just get rid of the macro entirely